source/content/guides/autopilot/03-enable-autopilot.md

```diff
@@ -25,20 +25,30 @@ Autopilot can be enabled for individual sites within each eligible Workspace.
 
 1. Click the <i className="fa fa-robot"></i>**Autopilot** icon in the Global Primary Navigation.
 
-1. View the list of sites for which Autopilot is available in the **Site** column of the **Autopilot Sites** table.
+1. Under **Autopilot Sites**, click the link under **READY TO DEPLOY**
```

Comment:
Remove this item and image referring to "ready to deploy". This is related to a different PR we're working on I believe.